actors/evm/src/interpreter/output.rs

```diff
@@ -33,7 +27,7 @@ impl Debug for Output {
 
 /// Message status code.
 #[must_use]
-#[derive(Clone, Debug, Display, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
no longer encoding the status code into CBOR? So we are relying on `StatusCode` for that now?

Reply:
we dont ever send it on the wire, this was useless!
Also, we cant do that any more with thr ActorError variant...